tmt/steps/__init__.py

```diff
@@ -1599,6 +1611,21 @@ def _emit_key(key: str) -> None:
         for key in keys:
             _emit_key(key)
 
+    @functools.cached_property
+    def enabled_by_when(self) -> bool:
+        """ Check if the plugin is enabled by 'when' keyword """
+        fmf_context = Context(**self.step.plan._fmf_context)
```

Comment:
I had to jump to the top of the file to learn which context exactly `Context` was :( Since tmt has its own container for "fmf context", would it make sense to refer to this one using its full name, `fmf.Context`, rather than just `Context`? I'm worried it's puzzling.

Reply:
fixed in 3d099121